src/sentry/tasks/file.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+
+from sentry.tasks.base import instrumented_task
+from sentry.tasks.deletion import MAX_RETRIES
+
+
+@instrumented_task(
+    name='sentry.models.tasks.delete_file',
```

Comment:
I know this is arbitrary, but let's match it to what the import path would be now.

`sentry.tasks.file.delete_file`

Reply:
My bad, forgot to fix that typo 😆